example/src/main/java/com/stripe/example/activity/PollingActivity.java

```diff
@@ -40,7 +40,9 @@
 
     private static final String FUNCTIONAL_SOURCE_PUBLISHABLE_KEY =
             "pk_test_vOo1umqsYxSrP5UXfOeL3ecm";
-    private static final String RETURN_URL = "stripe://example";
+    private static final String RETURN_SCHEMA = "stripe://";
+    private static final String RETURN_HOST = "example";
```

Comment:
Yep -- the original example was of polling asynchronously, so it was "example". I could make it "async" to make things obvious.